frontstage/views/account/accept_transfer_survey.py

```diff
@@ -0,0 +1,119 @@
+            selected_business = get_business_by_id(business_id)
+            transfer_dict[selected_business[0]['name']] = {
+                'trading_as': selected_business[0]['trading_as'],
+                'surveys': business_surveys
+            }
+        return render_template('surveys/surveys-transfer/summary.html',
+                               transfer_dict=transfer_dict,
+                               batch_no=batch_number,
+                               transferred_by=transferred_by)
+
+    except ApiError as exc:
+        # Handle api errors
+        if exc.status_code == 409:
+            logger.info('Expired transfer survey email verification token', token=token, api_url=exc.url,
+                        api_status_code=exc.status_code)
+            abort(409)
+        elif exc.status_code == 404:
+            logger.warning('Unrecognised transfer survey email verification token', token=token, api_url=exc.url,
+                           api_status_code=exc.status_code)
+            abort(404)
+        else:
+            logger.info('Failed to verify transfer survey email', token=token, api_url=exc.url,
+                        api_status_code=exc.status_code)
+            raise exc
+
+
+@account_bp.route('/confirm-transfer-surveys/<batch>', methods=['GET'])
+def accept_transfer_surveys(batch):
+    """
+    Accept endpoint when a transfer survey summary is accepted
+    :param batch: batch number
+    :type batch: str
+    """
+    logger.info('Attempting to get batch number', batch_number=batch)
+    try:
+        response = party_controller.get_pending_surveys_batch_number(batch)
+        is_existing_user = _is_existing_account(response.json()[0]['email_address'])
+    except ApiError as exc:
+        logger.error('Failed to confirm transfer survey', status=exc.status_code, batch_number=batch)
+        raise exc
+    if is_existing_user:
+        return redirect(url_for('account_bp.accept_transfer_surveys_existing_account', batch=batch))
+    return redirect(url_for('register_bp.pending_surveys_register_enter_your_details', batch_no=batch,
+                            email=response.json()[0]['email_address'], is_transfer=True))
+
+
+@account_bp.route('/confirm-transfer-surveys/<batch>/existing-account', methods=['GET'])
+@jwt_authorization(request)
+def accept_transfer_surveys_existing_account(session, batch):
+    """
+    Accept redirect endpoint for accepting transfer surveys for existing account
+    :param session:
+    :type session:
+    :param batch: batch number
+    :type batch: str
+    """
+    logger.info('Attempting to confirm transfer surveys for existing account', batch_number=batch)
+    try:
+        party_controller.confirm_pending_survey(batch)
+    except ApiError as exc:
+        logger.error('Failed to confirm transfer survey for existing account', status=exc.status_code, batch_number=batch)
+        raise exc
```

Comment:
As above

Reply:
🎨

Reply:
I think it will be fed back as we are raising the error. Please correct me if I am wrong.